blog/posts/basics-taking-screenshots.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Taking screenshots
+date: 2020-06-23
+author: Giovanni Rago
+githubUser: ragog
+tags: 
+  - basics
+---
+
+Headless browsers are fully capable of taking screenshots, which can be very useful in troubleshooting failures or faulty scripts. Through the use of additional libraries and tools, it is also possible to automate visual comparisons using 
```

Comment:
"which can be" => "which is"

Don't weaken your message

Reply:
"Through the use of" => "Using"

Too complex.

Reply:
Implementing.